### PR TITLE
GitHub Desktop 2.2.4 release

### PR DESCRIPTION
**Pull request by niik**
Release PR for GitHub Desktop 2.2.4.

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+      "[Fixed] Changed wording to match git autocrlf behavior - #5741. Thanks @jmhammock!",
+      "[Fixed] Don't dismiss dialogs when dragging from the dialog to the backdrop - #8544",
+      "[Fixed] Fix arrow key navigation of Changes and History lists (#6845) - #6845. Thanks @JMcKiern!",
+      "[Fixed] Handle multibyte characters in names - #8534",
+      "[Fixed] Keep PR badge on top of progress bar - #8622",
+      "[Fixed] Make newly cloned repositories load pull requests correctly - #8745",
+      "[Fixed] Prevent application crash when background process spawn fails - #8614",
+      "[Fixed] Show autocomplete suggestions on top of CodeMirror author input - #4084",
+      "[Fixed] Standardize dialogs' button orders - #7186 #2463 #5883",
+      "[Improved] Don't prompt to stash changes when switching away from protected branch - #8664",
```
Comment (nerdneha) [inline changelog.json]: Do we need this line given that protected branches is new?

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
   "releases": {
+    "2.2.4": [
+      "[New] Warn when committing to a protected branch - #7023",
+      "[Added] Adding integration for Xcode as external editor - #8255. Thanks @msztech!",
+      "[Fixed] Changed wording to match git autocrlf behavior - #5741. Thanks @jmhammock!",
+      "[Fixed] Don't dismiss dialogs when dragging from the dialog to the backdrop - #8544",
+      "[Fixed] Fix arrow key navigation of Changes and History lists (#6845) - #6845. Thanks @JMcKiern!",
+      "[Fixed] Handle multibyte characters in names - #8534",
+      "[Fixed] Keep PR badge on top of progress bar - #8622",
+      "[Fixed] Make newly cloned repositories load pull requests correctly - #8745",
```
Comment (niik) [inline changelog.json]:
```suggestion
```

This was a bug that was only present on beta (it was introduced by #8635) so I don't think it should be part of the release notes for prod.

File: changelog.json (hunk 1 above)
Comment (niik) [inline changelog.json]: Nope, suggest we remove it 👇 
```suggestion
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+      "[Fixed] Changed wording to match git autocrlf behavior - #5741. Thanks @jmhammock!",
+      "[Fixed] Don't dismiss dialogs when dragging from the dialog to the backdrop - #8544",
+      "[Fixed] Fix arrow key navigation of Changes and History lists (#6845) - #6845. Thanks @JMcKiern!",
+      "[Fixed] Handle multibyte characters in names - #8534",
+      "[Fixed] Keep PR badge on top of progress bar - #8622",
+      "[Fixed] Prevent application crash when background process spawn fails - #8614",
+      "[Fixed] Show autocomplete suggestions on top of CodeMirror author input - #4084",
+      "[Fixed] Standardize dialogs' button orders - #7186 #2463 #5883",
+      "[Improved] Make no newline indicator in file diffs readable by screenreaders. - #6494. Thanks @jmhammock!",
+      "[Improved] Warn user if they're committing to a repo they don't have write permission for - #8665"
```
Comment (nerdneha) [inline changelog.json]: Last thing, so sorry! I just saw this, should we make this [New]? It feels analogous to the protected branches one based on the way it’s written

Comment (niik) [reply changelog.json]: Nothing to apologize for! In fact I think this is also a good opportunity to look at the language one more time. How about...

```suggestion
      "[New] Warning when committing to a repository you don't have write access to - #8665"
```

Comment (nerdneha) [reply changelog.json]:
```suggestion
      "[New] Warn when committing to a repository you don't have write access to - #8665"
```

**Comment by niik** [comment]
This has shipped!

**Pull request merged**